### PR TITLE
Fix the installation instructions in the Elisp snippet

### PR DESCRIPTION
**Pull request by turbopape**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ or if you'd rather keep it in your dotfiles:
 
 ```el
 (unless (package-installed-p 'inf-clojure)
-  (package-refresh-contents))
+  (package-install 'inf-clojure))
```
Comment (bbatsov) [inline README.md]: Actually both lines should be here. I forgot the second line.

**Comment by bbatsov** [comment]
In the commit message:
* Fixed -> fix
* ELisp -> Elisp

**Comment by turbopape** [comment]
Done.

**Comment by bbatsov** [comment]
I see two commits now - they should be squashed.

**Comment by turbopape** [comment]
Done.

**Comment by bbatsov** [comment]
Thanks!

**Comment by turbopape** [comment]
You're welcome!

**Pull request merged**